src/rules/trailingCommaRule.ts

```diff
@@ -219,7 +219,7 @@ class TrailingCommaWalker extends Lint.AbstractWalker<Options> {
 
     /* Expects `list.length !== 0` */
     private checkComma(hasTrailingComma: boolean | undefined, list: ts.NodeArray<ts.Node>, closeTokenPos: number, optionKey: OptionName) {
-        const options = isSameLine(this.sourceFile, list[list.length - 1].end, closeTokenPos)
+        const options = isSameLine(this.sourceFile, list[list.length - 1].end, closeTokenPos) || this.isWrappedSingleExpression(list)
```

Comment:
If we really consider this a bug fix, that's probably ok. Otherwise this is a breaking change, because the rule now works exactly the other way around for such cases.
One possibility is to add an option to explicitly enable this new behavior.